### DISCUSSION TITLE
[Feature] is kafka-trace-exporter module plan to filter before send

### DISCUSSION THREAD

**Discussion by yqjdcyy** [apache/skywalking#10717] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

- kafka-trace-exporter module is great to extend the scenarioes of users
- sometime we may just need to `focus on exception`, and large of pods/ service means to large message of trace, but all messages require more and more support (kafka/ consumer of pod)
- one simple switch, maybe more useful than custom filters like oal, will be a great helpful

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
I think we don't have things similar to this.
If you want to contribute this to upstream, please write a proposal for review.
BTW, OAL is for analysis, rather than filtering the traces.
